api/pkg/apis/v1alpha1/providers/target/helm/helm.go

```diff
@@ -510,7 +511,10 @@ func (i *HelmTargetProvider) Apply(ctx context.Context, deployment model.Deploym
 				instance:  deployment.Instance,
 				populator: i.MetaPopulator,
 			}
-			installClient := configureInstallClient(component.Component.Name, &helmProp.Chart, &deployment, actionConfig, postRender)
+			installClient, err := configureInstallClient(component.Component.Name, &helmProp.Chart, &deployment, actionConfig, postRender)
```

Comment:
How about upgrade and uninstall client? Helm upgrade / Helm uninstall also support --wait and --timeout

Reply:
We need to add uninstall function to this target provider?

Reply:
The provider already supports uninstallation (without the switches)

Reply:
Thanks haishi, I will fix it in next pr.